flocker/control/_protocol.py

```diff
@@ -240,6 +243,19 @@ class ClusterStatusCommand(Command):
     response = []
 
 
+class SetNodeEraCommand(Command):
+    """
+    Set the era for a node.
+
+    This should clear any previous NodeState that has a different
+    era. Updates to the node should only be sent after this command, to
+    ensure it doesn't get stale pre-reboot information (i.e. NodeState
+    with wrong era).
```

Comment:
Yep. See https://clusterhq.atlassian.net/browse/FLOC-3380, and parent issue for the full plan.